pylama.ini

```diff
@@ -28,7 +28,7 @@ ignore = R0911
 # R0911 Too many return statements (8/6) [pylint]
 
 [pylama:cloudmarker/alerts/emailalert.py]
-ignore = R0913,C0412
+ignore = R0913,C0412,R0902
```

Comment:
What do these ignore? Please add comments below this `ignore` line for each warning being ignored. See the rest of the file for these commenting conventions.

Reply:
https://github.com/cloudmarker/cloudmarker/commit/11ff34083ba27900832dc7daf05a72cd2a99caca#diff-21c09aff342003d14df85a14eeaae393R33